### ISSUE TITLE
A custom div not working with the voila 

### ISSUE THREAD

**Issue by dou-du** [voila-dashboards/voila#469]
Hi,

We are working on a Jupyter extension to reprint the code easily. Basically, we need
put a custom <div> generated by the Markdown. (as shown below for the second 
display). 

![Screenshot from 2019-11-07 12-54-22](https://user-images.githubusercontent.com/56022756/68387393-7d5caa80-015e-11ea-8fda-ae6e335ba520.png)

However, the voila is not working for our second display. The results is shown in figure 
below:

![Screenshot from 2019-11-07 12-55-11](https://user-images.githubusercontent.com/56022756/68387476-b3019380-015e-11ea-92d0-d51c7e99306f.png)

Is there a way to solve this problem ? Thanks a lot for helping.

**Comment by giovannipizzi** [comment]
Indeed, it would be great to have a pointer of what could be the problem - we would be happy to contribute with a PR if some developer could point us to the point in the code where we should look into.
As a reference, the full widget is [reprint-magic](https://github.com/osscar-org/reprint-magic) - the example above by @dou-du is the minimal example to illustrate the issue we are facing. Thanks!
